spec/unit/enforcement_spec.rb

```diff
@@ -1,15 +1,29 @@
 # frozen_string_literal: true
+class MyController # < ApplicationController
```

Comment:
@atz The general practice has been for each spec to clean up after itself so that other specs run the same way if this test was run with it or not.  I believe all @jcoyne is asking for is for you to do is `Object.send(:remove_const, :MyController)` after the test